sentinel-core/src/main/java/com/alibaba/csp/sentinel/util/ConfigUtil.java

```diff
@@ -0,0 +1,119 @@
+package com.alibaba.csp.sentinel.util;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Enumeration;
+import java.util.List;
+import java.util.Properties;
+
+/**
+ * <p>
+ * util for loading config
+ * </p>
+ *
+ * @author lianglin
+ * @since 1.7.0
+ */
+public final class ConfigUtil {
+
+
+    /**
+     * Return null if the file not exist
+     *
+     * @param fileName
+     * @return
+     */
+    public static Properties loadPropertiesFromFile(String fileName) {
+        if (StringUtil.isNotBlank(fileName)) {
+            if (fileName.startsWith("/")) {
+                return loadPropertiesFromAbsoluteFile(fileName);
+            } else {
+                return loadPropertiesFromRelativeFile(fileName);
+            }
+        } else {
+            return null;
+        }
+    }
+
+    private static Properties loadPropertiesFromAbsoluteFile(String fileName) {
+        Properties properties = null;
+        try {
+
+            File file = new File(fileName);
+            if (!file.exists()) {
+                return null;
+            }
+
+            FileInputStream input = new FileInputStream(file);
+            try {
+                properties = new Properties();
+                properties.load(input);
+            } finally {
+                input.close();
+            }
+        } catch (Throwable e) {
+            e.printStackTrace();
+        }
+        return properties;
+    }
+
+
+    private static Properties loadPropertiesFromRelativeFile(String fileName) {
+        List<URL> list = new ArrayList<URL>();
+        try {
+            Enumeration<URL> urls = getClassLoader().getResources(fileName);
```

Comment:
I've tested the code in several scenarios. For directly run the program in IDE (i.e. `java xxx`), it could work well. However, when I tested this with a fat-jar, the relative path seemed not to work well.

For example, a jar is located in `temp` directory and the config file is located in `temp/config/my-config.properties`. In `temp` directory I execute `java -Dcsp.sentinel.config.file=config/my-config.properties -jar test.jar`, and Sentinel will not retrieve properties from the target file.

Reply:
try` java -Dcsp.sentinel.config.file= my-config.properties -jar test.jar` ?